### PR TITLE
Fix/anthos bug fixes

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/anthos`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-7612, #MANAGER-7613, #MANAGER-7669, #MANAGER-7688
| License          | BSD 3-Clause

## Description

-> Remove commitment
-> Fix storage information in add storage popup
-> Remove delete option for 'Active' private IPs
-> Update description for delete host popup
-> Fix tracking bugs
-> Remove netapp access from security tile

### CODE REVIEW

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html [hunk 1]
```diff
@@ -16,8 +17,14 @@
                     data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_information_description' | translate }}"
                     data-description="{{:: $ctrl.tenant.description }}"
                 >
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.goToRenameService()">
+                    <oui-action-menu
+                        aria-label="Actions"
```
Comment (antleblanc) [inline packages/manager/modules/anthos/src/dashboard/general-information/general-information.html:21]: **nitpick:** Might be great if we can move this to a translation file.

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html [hunk 2]
```diff
@@ -52,64 +64,84 @@
...
                         </a>
                     </oui-tile-description>
 
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onAnthosRegeneratePasswordClick()">
-                                <span
-                                    data-translate="anthos_tenant_dashboard_general_information_tile_security_center_access_anthos_menu_regenerate_pwd"
-                                ></span>
+                    <oui-action-menu
+                        aria-label="Actions"
```
Comment (antleblanc) [inline packages/manager/modules/anthos/src/dashboard/general-information/general-information.html:92]: **nitpick:** Might be great if we can move this to a translation file.

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html [hunk 3]
```diff
@@ -52,64 +64,84 @@
...
                         </a>
                     </oui-tile-description>
 
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onNetappRegeneratePasswordClick()">
-                                <span
-                                    data-translate="anthos_tenant_dashboard_general_information_tile_security_center_access_netapp_menu_regenerate_pwd"
-                                ></span>
+                    <oui-action-menu
+                        aria-label="Actions"
```
Comment (antleblanc) [inline packages/manager/modules/anthos/src/dashboard/general-information/general-information.html:132]: **nitpick:** Might be great if we can move this to a translation file.

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html [hunk 4]
```diff
@@ -138,9 +163,16 @@
             >
                 <!--Hosts-->
                 <oui-tile-definition
-                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_resources_hosts' | translate }}">
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onGoToOrderHost()">
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_resources_hosts' | translate }}"
+                >
+                    <oui-action-menu
+                        aria-label="Actions"
```
Comment (antleblanc) [inline packages/manager/modules/anthos/src/dashboard/general-information/general-information.html:169]: **nitpick:** Might be great if we can move this to a translation file.

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html [hunk 5]
```diff
@@ -160,43 +194,19 @@
...
-                        </p>
+                        <anthos-storage-usage
+                            data-storage-usage="$ctrl.storageUsage"
+                        ></anthos-storage-usage>
                     </oui-tile-description>
 
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onGoToOrderStorage()">
+                    <oui-action-menu
+                        aria-label="Actions"
```
Comment (antleblanc) [inline packages/manager/modules/anthos/src/dashboard/general-information/general-information.html:203]: **nitpick:** Might be great if we can move this to a translation file.

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html [hunk 6]
```diff
@@ -206,14 +216,23 @@
 
                 <!--IPs-->
                 <oui-tile-definition
-                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_resources_ips' | translate }}">
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onGoToOrderPublicIp()">
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_resources_ips' | translate }}"
+                >
+                    <oui-action-menu
+                        aria-label="Actions"
```
Comment (antleblanc) [inline packages/manager/modules/anthos/src/dashboard/general-information/general-information.html:222]: **nitpick:** Might be great if we can move this to a translation file.

File: packages/manager/modules/anthos/src/dashboard/host/remove/translations/Messages_fr_FR.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "anthos_dashboard_remove_host_title": "Supprimer votre hôte",
-  "anthos_dashboard_remove_host_description": "Cette machine sera immédiatement supprimée, et vous ne serez plus facturé au-delà de sa date de fin d’engagement (le {{expiryDate}}).",
+  "anthos_dashboard_remove_host_description": "Cette machine sera immédiatement supprimée, et vous ne serez plus facturé.",
```
Comment (antleblanc) [inline packages/manager/modules/anthos/src/dashboard/host/remove/translations/Messages_fr_FR.json:3]:
```suggestion
  "anthos_dashboard_remove_host_description": "Cette machine sera immédiatement supprimée et vous ne serez plus facturé.",
```

**Review by antleblanc — CHANGES_REQUESTED** [review]
Nice work @ganeshkumar1989!

I noticed that some template still have untranslated content. Especially some labels.
Could you please refer to a translation instead?

**Comment by ganeshkumar1989** [comment]
> Nice work @ganeshkumar1989!
> 
> I noticed that some template still have untranslated content. Especially some labels.
> Could you please refer to a translation instead?

Hello @antleblanc . Thanks for your feedback. This has been handled now

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html (hunk 1 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/general-information/general-information.html]: This is done

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html (hunk 2 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/general-information/general-information.html]: This is done

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html (hunk 3 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/general-information/general-information.html]: This is done

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html (hunk 4 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/general-information/general-information.html]: This is done

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html (hunk 5 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/general-information/general-information.html]: This is done

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.html (hunk 6 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/general-information/general-information.html:222]: This is done

File: packages/manager/modules/anthos/src/dashboard/host/remove/translations/Messages_fr_FR.json (hunk 7 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/host/remove/translations/Messages_fr_FR.json]: This is fixed

File: packages/manager/modules/anthos/src/dashboard/general-information/order-host/order-host.routing.js [hunk 8]
```diff
@@ -14,5 +14,8 @@ export default /* @ngInject */ ($stateProvider) => {
         return 'order-host';
       },
     },
+    atInternet: {
+      rename: 'hpc::anthos::project::order-host',
```
Comment (radireddy) [inline packages/manager/modules/anthos/src/dashboard/general-information/order-host/order-host.routing.js:18]: may be you extract common, duplicated prefix "hpc::anthos::project::" into constants or route file?

Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/general-information/order-host/order-host.routing.js]: This is done. Thanks

**Review by radireddy — APPROVED** [review]

**Review by varun257 — APPROVED** [review]

**Pull request merged**